### ISSUE TITLE
Update PMO and the PMO Guide to not use can-connect/can/tag

### ISSUE THREAD

**Issue by phillipskevin** [donejs/donejs#803]
From https://github.com/donejs/donejs/issues/515
- [x] once merged, update generated models to match donejs/generator-donejs#197 (don't import or call can-connect/can/tag, add algebra, don't set idProp, etc)
- [x] change the restaurant model (`section_Createadditionalcomponents`) to load data in the viewModel
- [x] update the guide (`section_Addingreal_timeeventstoamodel`) to import `can-connect/can/tag/` in the model and explain that this can be used to load data directly in the template

**Comment by phillipskevin** [comment]
closed by https://github.com/donejs/donejs/issues/817.

**Comment by phillipskevin** [comment]
Missed one change that needed to be made. The `Order` model should use an enum for the statuses so requests can be combined, like: 

```
set.comparators.enum("status", ["new", "preparing", "delivery", "delivered"])
```

**Comment by phillipskevin** [comment]
Closed by #831.
